src/accelerate/commands/config/config_args.py

```diff
@@ -60,7 +60,7 @@ def load_config_from_file(config_file):
                 config_class = ClusterConfig
             else:
                 config_class = SageMakerConfig
-            return config_class.from_json_file(json_file=config_file)
+                return config_class.from_json_file(json_file=config_file)
```

Comment:
Hmm, can you explain this change? To me, it looks like we're in the `if config_file.endswith(".json")` case. If we don't return here, the `config_class` will be called with `.from_yaml_file` farther below.

Reply:
That would be a typo when I was trying to catch at the right time, will revert.